src/Attr.cc

```diff
@@ -456,6 +458,17 @@ void Attributes::CheckAttr(Attr* a)
 		break;
 
 	case ATTR_EXPIRE_READ:
+		{
+		if ( Find(ATTR_BROKER_STORE) )
+			{
+			Error("&broker_store and &read_expire cannot be used simultaneously");
+			}
+		if ( Find(ATTR_BACKEND) )
+			{
+			Error("&backend and &broker_store cannot be used simultaneously");
```

Comment:
I suppose this should be saying `... and &read_expire`

Reply:
Yes. I thought I fixed that one, sorry.